src/Models/AzureComponentConfig.ts

```diff
@@ -47,31 +47,29 @@ export class AzureConfigFileHandler {
         AzureComponentsStorage.fileName);
   }
 
-  createIfNotExists() {
+  async createIfNotExists(type: ScaffoldType) {
     const azureConfigs: AzureConfigs = {componentConfigs: []};
     const azureConfigFolderPath =
         path.join(this.projectRootPath, AzureComponentsStorage.folderName);
-    if (!fs.existsSync(azureConfigFolderPath)) {
+    if (!await FileUtility.exists(type, azureConfigFolderPath)) {

```

Comment:
use dir exists?

Reply:
Remove the `exists` api and fix this issue.